src/coreclr/src/tools/Common/System/EnumExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
```

Comment:
I can see that you removed usage of these apis, but this file seems to still be here. Please remove this file.

Reply:
Oh oops, yes.